packages/calcite-components/src/components/input-time-zone/input-time-zone.e2e.ts

```diff
@@ -36,10 +36,22 @@ describe("calcite-input-time-zone", () => {
 
   // for stability, we use time zones that are unaffected by daylight savings time
   const testTimeZoneItems: TestTimeZoneItem[] = [
-    { name: "America/Mexico_City", offset: -360, label: "GMT-6" },
-    { name: "America/Phoenix", offset: -420, label: "GMT-7" },
-    { name: "Pacific/Guam", offset: 600, label: "GMT+10" },
-    { name: "Pacific/Galapagos", offset: -360, label: "GMT-6" },
+    {
+      name: "America/Mexico_City",
+      offset: -360,
+      label: "(GMT-6) Bahia de Banderas, Cambridge Bay, Costa Rica, El Salvador, Managua, Monterrey, and Tegucigalpa",
```

Comment:
FIXME: we’ll need to revert this and apply the [suggested changes](https://github.com/Esri/calcite-design-system/pull/12252#discussion_r2116186914) to region-mode tests only. The labels in offset mode vary with time zone changes throughout the year, so those tests will eventually fail when any of the related regions enter or exit DST. The comment above this variable provides a bit of context. I’ll update it to clarify further.